test/runtests.jl

```diff
@@ -61,9 +61,23 @@ end
                 @test neighbors(Y) == k
                 @test length(vertices(Y)) > 1
             end
+            
+            # test if we provide pre-computed Gram matrix
+            if algorithm == DiffMap && T == Float64
```

Comment:
Why do you test only with  `T == Float64`? Try to remove it.

Reply:
done! tests pass for `Float64` too.
(sorry, just didn't see the point of making tests for different data points honestly. never used `Float32`'s before!)